src/app/nice.py

```diff
@@ -0,0 +1,53 @@
+# -----------------------------------------
+# Nice and dry Phantom App
+# -----------------------------------------
+# TODO: This SHOULD be split into its own repo and imported
```

Comment:
Maybe we put this in a new repo next to the `deploy.py` script.

Reply:
We need to think of a repo name, but I like the idea of packaging the helper script and the deploy.py script.